core/domain/topic_services.py

```diff
@@ -826,6 +832,8 @@ def save_topic_summary(topic_summary):
         'uncategorized_skill_count': topic_summary.uncategorized_skill_count,
         'subtopic_count': topic_summary.subtopic_count,
         'total_skill_count': topic_summary.total_skill_count,
+        'total_published_node_count':
```

Comment:
Why not ``total_published_chapter_count``, instead?

Reply:
In the backend Python code, we refer to chapters as nodes (for some reason). For consistency, I used `node_count`.